external-builds/jax/README.md

```diff
@@ -0,0 +1,174 @@
+| ----------------- | ------------- | ---------------- |
+| jaxlib            | ✅ Supported  | ❌ Not supported |
+| jax_rocmX_pjrt    | ✅ Supported  | ❌ Not supported |
+| jax_rocmX_plugin  | ✅ Supported  | ❌ Not supported |
+
+### Supported JAX versions
+
+We support building various Jax versions compatible with the latest ROCm
+sources and release packages.
+
+Support for JAX is provided via the stable release branch of [ROCm/rocm-jax](https://github.com/ROCm/rocm-jax). Can build using the stable release branch (for example, `rocm-jaxlib-v0.8.0`) to suit your requirements.
+
+See the following table for supported version:
+
+| JAX version | Linux                                                                                                                                   | Windows          |
+| ----------- | --------------------------------------------------------------------------------------------------------------------------------------- | ---------------- |
+| 0.8.0       | ✅ Supported<br><ul><li>[ROCm/rocm-jax `rocm-jaxlib-v0.8.0` branch](https://github.com/ROCm/rocm-jax/tree/rocm-jaxlib-v0.8.0)</li></ul> | ❌ Not supported |
+
+## Build instructions
+
+This repository builds the ROCm-enabled JAX artifacts:
+
+- jaxlib (ROCm)
+- jax_rocmX_pjrt (PJRT runtime for ROCm)
+- jax_rocmX_plugin (JAX runtime plugin for ROCm)
+
+We support simple flow. The path uses TheRock tarballs for ROCm install.
+
+### Prerequisites
+
+- **OS**: Linux (supported distributions with ROCm)
+- **Python**: 3.12 recommended
+- **Compiler**: clang (provided via TheRock tarball or system)
+
+### Steps
+
+- Checkout rocm-jax
+
+  - `git clone https://github.com/ROCm/rocm-jax.git`
+  - `cd rocm-jax`
+
+- Choose versions and TheRock Source
+
+  - Pick Python version
+  - Pick a TheRock tarball URL, a local tarball file path or a directory containing ROCm installation
+  - the TAR url paths for nightly `https://rocm.nightlies.amd.com/tarball/`
+
+- Build all wheels using a tarball URL:
+
+  ```bash
+  python3 build/ci_build \
+  --compiler=clang \
+  --python-versions="3.12" \
+  --rocm-version="<rocm_version>" \
+  --therock-path="<tar.gz path>" \
+  dist_wheels
+  ```
+
+  example for `therock-path=https://rocm.nightlies.amd.com/tarball/therock-dist-linux-gfx94X-dcgpu-7.10.0a20251120.tar.gz`
+  `rocm_version=7.10.0a20251120`
+
+- Locate built wheels
+  After successfull build, wheel will be available in:
+  `jax_rocm_plugin/wheelhouse/`
+
+## Test instructions
+
+### Prerequisites
+
+- AMD GPU matching the target `amdgpu_family`
+- Python environment with pip
+- Access to the JAX wheel package index (staging)
+
+### Local Testing with Built Wheels Steps
+
+After building wheels locally, you can test them:
+
+- Checkout TheRock Repo
+
+  ```bash
+  git clone https://github.com/ROCm/TheRock.git
+  cd TheRock
+  ```
+
+- Checkout rocm-jax repo
+
+  - `git clone https://github.com/rocm/rocm-jax.git jax`
+  - `cd jax`
+  - `git checkout rocm-jaxlib-v<JAX_VERSION>`
```

Comment:
Why isn't this a code block as the one above? Furthermore, this will result in cloning rocm-jax within `TheRock` if following the instructions step by step which might not be desired (or is it).